test/verify/check-machines

```diff
@@ -473,6 +473,55 @@ class TestMachines(MachineCase):
 
         b.wait_in_text("#vm-subVmTest1-network-2-state span", "up")
 
+    def testVCPU(self):
+        b = self.browser
+        m = self.machine
+
+        self.startVm("subVmTest1")
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+        b.click("tbody tr th") # click on the row header
+
+        b.wait_present("#vm-subVmTest1-state")
+        b.wait_in_text("#vm-subVmTest1-state", "running")
```

Comment:
could this test the behaviour of clicking on the vcpus-count link while the machine is running? I suppose it should either not be clickable at all, or show the dialog with read-only values?

Reply:
Actually VCPU settings can be changed in runing VM, but all changes will take effect only after reboot.

Reply:
Might make sense to put that into the dialog as an info bar? Or, to avoid confusion, only allow changing it when not running?

Reply:
I think variant with info bar make more sense, because delayed VCPU changing can be useful.